DataSourceKit/CollectionViewDataSource.swift

```diff
@@ -8,17 +8,19 @@
 
 import UIKit
 
-public class CollectionViewDataSource<CellDeclaration>: NSObject, UICollectionViewDataSource {
+open class CollectionViewDataSource<CellDeclaration>: NSObject, UICollectionViewDataSource {
```

Comment:
I think this is not meant to be `overridden` or `subclassed` outside the module.